crates/bevy_render/src/camera/projection.rs

```diff
@@ -266,3 +266,84 @@ impl Default for OrthographicProjection {
         }
     }
 }
+
+impl OrthographicProjection {
+    /// Create a new `OrthographicProjection` with default values.
+    ///
+    /// Use its other methods to modify its properties, or modify them after creation if `mut`.
+    ///
+    /// ```
+    /// use bevy_render::camera::{OrthographicProjection, ScalingMode};
+    /// let projection = OrthographicProjection::new()
+    ///     .scale(1.0)
+    ///     .clipping_plane(0.0, 1000.0)
+    ///     .viewport_origin(0.5, 0.5)
+    ///     .scaling_mode(ScalingMode::WindowSize(1.0));
+    /// ```
+    pub fn new() -> Self {
```

Comment:
Should there be a separate builder type? This pseudo builder pattern feels sufficient.